example/java/prepare.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 CURDIR="$(cd "$(dirname "$0")"; pwd)"
 IROHA_HOME="$(dirname $(dirname "${CURDIR}"))"
-cmake -H$IROHA_HOME -Bbuild -DSWIG_JAVA=ON -DSWIG_JAVA_PKG="jp.co.soramitsu.iroha";
+cmake -H$IROHA_HOME -Bbuild -DSWIG_JAVA=ON -DSWIG_JAVA_PKG="jp.co.soramitsu.iroha" -GNinja
```

Comment:
I don't think that every script user has Ninja installed, so it seems better to leave the default value.